app/views/layouts/application.html.erb

```diff
@@ -24,6 +24,9 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+<!--[if ( !IE ) | ( gte IE 9 ) ]><!-->
+<%# render 'dough/shared/social_sharing_icon_sprite' %>
```

Comment:
Yeah I know that, but just seems the wrong way around for progressive enhancement. Most of the enhancements for newer browsers will just get ignored in <IE9 so we don't have to specifically write conditional comments for everything. I was wondering if it actually causes any issues in IE8 and 9?